proto/cmp/services/activity/v1alpha/activity_types.proto

```diff
@@ -0,0 +1,200 @@
+syntax = "proto3";
+
+// ### Activity Services
+//
+// The Activity services are used for both tickets and excursions
+
+package cmp.services.activity.v1alpha;
+
+import "cmp/types/v1alpha/description.proto";
+import "cmp/types/v1alpha/location.proto";
+import "cmp/types/v1alpha/language.proto";
+import "cmp/types/v1alpha/file.proto";
+import "cmp/types/v1alpha/datetime_range.proto";
+import "cmp/types/v1alpha/duration.proto";
+import "cmp/types/v1alpha/contact_info.proto";
+import "cmp/types/v1alpha/product_code.proto";
+import "cmp/types/v1alpha/delivery.proto";
+import "cmp/types/v1alpha/redemption.proto";
+import "cmp/services/activity/v1alpha/search_result_types.proto";
+
+message ActivityExtendedInfo {
```

Comment:
If this is an extended info message, there should be a version of a normal info message, right? 

Similar to `Property` and `PropertyExtendedInfo` messages. The extended info message contains a link to the property message (so it does not repeat the similar fields it has).

And we have a "normal" message for this named `Activity` in the `search_results_types.proto`. Is there a reason we can not link or use it here?

Reply:
ActivityExtendedInfo has more information than Activity , We can extend Activity message with more info if we want.